src/main/webapp/app/assessment/assessment-header/assessment-header.component.html

```diff
@@ -144,7 +144,12 @@ <h3 class="top-container flex-wrap flex-lg-nowrap">
         <br />
         <div class="d-flex flex-wrap flex-lg-nowrap">
             <ng-container *ngIf="!result?.completionDate">
-                <button class="btn m-1 btn-primary" (click)="save.emit()" [disabled]="!assessmentsAreValid || !isAssessor || saveBusy || submitBusy || cancelBusy">
+                <button
+                    class="btn m-1 btn-primary"
+                    (click)="save.emit()"
+                    [disabled]="!assessmentsAreValid || !isAssessor || saveBusy || submitBusy || cancelBusy"
+                    [ngbTooltip]="'Ctrl S'"
```

Comment:
```suggestion
                    [ngbTooltip]="'Ctrl+S'"
```
The usual convention is that keyboard shortcuts are shown with `+` between the keys instead of spaces.